lib/util/ModelUtil.js

```diff
@@ -22,4 +22,15 @@ export function is(element, type) {
  */
 export function getBusinessObject(element) {
   return (element && element.businessObject) || element;
+}
+
+/**
+ * Return the di object for a given element.
+ *
+ * @param  {djs.model.Base|ModdleElement} element
```

Comment:
This should not be an instance of `ModdleElement` anymore, right? Otherwise that would be a miss-use of the API.

Reply:
Good catch, I fixed it :heavy_check_mark: